system/Debug/Exceptions.php

```diff
@@ -372,17 +372,23 @@ protected function determineCodes(Throwable $exception): array
 	 */
 	public static function cleanPath(string $file): string
 	{
-		if (strpos($file, APPPATH) === 0)
+		switch (true)
 		{
-			$file = 'APPPATH/' . substr($file, strlen(APPPATH));
-		}
-		elseif (strpos($file, SYSTEMPATH) === 0)
-		{
-			$file = 'SYSTEMPATH/' . substr($file, strlen(SYSTEMPATH));
-		}
-		elseif (strpos($file, FCPATH) === 0)
-		{
-			$file = 'FCPATH/' . substr($file, strlen(FCPATH));
+			case strpos($file, VIEWPATH) === 0:
+				$file = 'VIEWPATH' . DIRECTORY_SEPARATOR . substr($file, strlen(VIEWPATH));
```

Comment:
Unfortunately, we cannot assume the view path will always be what is set in the config file. When using themes, it's standard to grab a new instance of the renderer while setting a different base path. So no good way to clean this one up. Please remove.